FinSearchUnified/Helper/UrlBuilder.php

```diff
@@ -19,7 +19,7 @@ class UrlBuilder
     const JSON_CONFIG = '/config.json';
     const ALIVE_ENDPOINT = 'alivetest.php';
     const SEARCH_ENDPOINT = 'index.php';
-    const NAVIGATION_ENPOINT = 'selector.php';
+    const NAVIGATION_ENDPOINT = 'selector.php';
```

Comment:
Okay, wow. I've been working with this code over a year now and never noticed this. Nice catch!